src/app/auth/auth.component.ts

```diff
@@ -1,35 +1,46 @@
 import { Component, OnInit } from '@angular/core';
-import {Collegue} from "./auth.domains";
-import {AuthService} from "./auth.service";
-import {Router} from "@angular/router";
+import { Collegue } from "./auth.domains";
+import { AuthService } from "./auth.service";
+import { Router } from "@angular/router";
 
 /**
  * Formulaire d'authentification.
  */
 @Component({
   selector: 'app-auth',
   templateUrl: './auth.component.html',
-  styles: []
+  styles: ['./auth.component.css']
 })
 export class AuthComponent implements OnInit {
 
 
-  collegue:Collegue = new Collegue({});
-  err:boolean;
+  collegue: Collegue = new Collegue({});
+  err: boolean;
 
-  constructor(private _authSrv:AuthService, private _router:Router) { }
+  constructor(private _authSrv: AuthService, private _router: Router) { }
 
   ngOnInit() {
   }
 
   connecter() {
     this._authSrv.connecter(this.collegue.email, this.collegue.motDePasse)
       .subscribe(
-        // en cas de succès, redirection vers la page /tech
+        // en cas de succès, redirection vers la page d'accueil
+        // TODO
         col => this._router.navigate(['/tech']),
 
         // en cas d'erreur, affichage d'un message d'erreur
-        err =>this.err = true
+        err => this.err = true
```

Comment:
Si vous n'utilisez pas le err, la notation correcte est la suivante: () => this.err = true